### PR TITLE
Update file outdated korean docs in dev-1.16-ko.4

### PR DESCRIPTION
**Pull request by ysyukr**
### **14 out of 20 files to be modified**
- [x] content/ko/docs/concepts/architecture/cloud-controller.md
- [x] content/ko/docs/concepts/cluster-administration/federation.md
- [x] content/ko/docs/concepts/overview/working-with-objects/names.md
> 변경점이 많아 재번역을 진행했습니다.
- [x] content/ko/docs/concepts/workloads/controllers/deployment.md
- [x] content/ko/docs/reference/glossary/applications.md
- [x] content/ko/docs/reference/glossary/replica-set.md
> 변경점이 많아 재번역을 진행했습니다.
- [x] content/ko/docs/reference/kubectl/cheatsheet.md
- [x] content/ko/docs/setup/_index.md
- [x] content/ko/docs/setup/best-practices/certificates.md
> 변경점에 대한 번역 진행
- [x] content/ko/docs/setup/best-practices/node-conformance.md
- [x] content/ko/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/declarative-config.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/imperative-config.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

### **Unmodified document and reason.**
- [x] content/ko/docs/concepts/architecture/controller.md
- [x] content/ko/docs/concepts/workloads/pods/pod-overview.md
- [x] content/ko/docs/reference/glossary/cri.md 
- [x] content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md
- [x] content/ko/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md
> #17059 에서 반영
- [x] content/ko/examples/controllers/daemonset.yaml
> #16962 에서 반영

from #17187
/language ko

### CODE REVIEW

File: content/ko/docs/concepts/overview/working-with-objects/names.md [hunk 1]
```diff
@@ -6,11 +6,13 @@ weight: 20
 
 {{% capture overview %}}
 
-쿠버네티스 REST API의 모든 오브젝트는 이름과 UID로 명백히 식별된다.
+클러스터의 각 오브젝트에는 해당 유형의 리소스에 고유한 [_이름_](#names) 을 가지고 있다.
+또한, 모든 쿠버네티스 오브젝트에는 전체 클러스터에 걸쳐 고유한 [_UID_](#uids) 를 가지고 있다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
또한, 모든 쿠버네티스 오브젝트는 전체 클러스터에 걸쳐 고유한 [_UID_](#uids) 를 가지고 있다.
```
조금 자연스럽게 변경해보았습니다. 참고하시기 바랍니다.

File: content/ko/docs/concepts/overview/working-with-objects/names.md [hunk 2]
```diff
@@ -6,11 +6,13 @@ weight: 20
 
 {{% capture overview %}}
 
-쿠버네티스 REST API의 모든 오브젝트는 이름과 UID로 명백히 식별된다.
+클러스터의 각 오브젝트에는 해당 유형의 리소스에 고유한 [_이름_](#names) 을 가지고 있다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
클러스터의 각 오브젝트는 해당 유형의 리소스에 대하여 고유한 [_이름_](#names) 을 가지고 있다.
```
조금 자연스럽게 변경해보았습니다. 참고하시기 바랍니다.

File: content/ko/docs/concepts/overview/working-with-objects/names.md [hunk 3]
```diff
@@ -6,11 +6,13 @@ weight: 20
...
+또한, 모든 쿠버네티스 오브젝트에는 전체 클러스터에 걸쳐 고유한 [_UID_](#uids) 를 가지고 있다.
 
-유일하지 않은 사용자 제공 속성에 대해서, 쿠버네티스는 [레이블](/docs/user-guide/labels)과 [어노테이션](/docs/concepts/overview/working-with-objects/annotations/)을 제공한다.
+예를 들어, 이름이 “myapp-1234”인 파드는 하나만 가질 수 있지만, 각각 “myapp-1234”라는
```
Comment (seokho-son) [inline content/ko/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
예를 들어, 이름이 “myapp-1234”인 파드는 하나만 가질 수 있지만, 이름이 “myapp-1234”인
```

File: content/ko/docs/concepts/overview/working-with-objects/names.md [hunk 4]
```diff
@@ -6,11 +6,13 @@ weight: 20
...
+이름을 가진 하나의 파드와 디플로이먼트를 가질 수 있다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
파드와 디플로이먼트는 각각 가질 수 있다.
```
문장 구조를 조금 변경해보았습니다. 참고하시기 바랍니다.

File: content/ko/docs/setup/best-practices/certificates.md [hunk 5]
```diff
@@ -79,6 +82,19 @@ etcd 역시 클라이언트와 피어 간에 상호 TLS 인증을 구현한다.
 | server | digital signature, key encipherment, server auth                                |
 | client | digital signature, key encipherment, client auth                                |
 
+{{< note >}}
+위에 나열된 호스트/SAN은 작업중인 클러스터를 획득하는데 권장된다. 특정 설정이 필요한 경우, 모든 서버 인증서에 SAN을 추가할 수 있다.
```
Comment (seokho-son) [inline content/ko/docs/setup/best-practices/certificates.md]:
```suggestion
위에 나열된 호스트/SAN은 작업 중인 클러스터를 획득하는데 권장된다. 특정 설정이 필요한 경우, 모든 서버 인증서에 SAN을 추가할 수 있다.
```

File: content/ko/docs/concepts/overview/working-with-objects/names.md [hunk 6]
```diff
@@ -38,8 +40,19 @@ spec:
...
 ## UID {#uids}
 
 {{< glossary_definition term_id="uid" length="all" >}}
 
+쿠버네티스 UID는 보편적으로 고유한 식별자이다(또는 UUID라고 한다).
+UUID는 ISO/IEC 9834-8 과 ITU-T X.667 로 표준화 되어 있다.
+
+{{% /capture %}}
+{{% capture whatsnext %}}
+* 쿠버네티스의 [레이블](/docs/concepts/overview/working-with-objects/labels/)에 대해 읽기.
```
Comment (seokho-son) [inline content/ko/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
* 쿠버네티스의 [레이블](/ko/docs/concepts/overview/working-with-objects/labels/)에 대해 읽기.
```
한글 문서가 확인되어 한글 문서 링크로 수정 제안 드립니다.

**Comment by ysyukr** [comment]
@seokho-son Thank you for the review.  I've checked and reflected your suggestions.

**Comment by seokho-son** [comment]
@ysyukr Thank you !
/lgtm
/approve

**Pull request merged**